tests/posix/headers/testcase.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   arch_exclude: posix
   tags: posix net socket
+  min_ram: 21
```

Comment:
This number is generally set to "usual" ram sizes. Next after 21 would be 32

Reply:
Done